test/directory.build.props

```diff
@@ -0,0 +1,3 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(SolutionDir)test\xdp.test.props" />
```

Comment:
Does this get imported before everything else?

Reply:
no it does not ☹️ 

it gets included by a microsoft.common.blah.props halfway through our .vcxproj files. anything we set here needs to be amenable to being included halfway through

Reply:
it might be included by a .props file included by Microsoft.Cpp.Default.props, but i already lost the documentation that explained where it gets included.

Reply:
yeah, it is included by Microsoft.Common.props, which we never directly include. i didn't detangle exactly where that occurs in our build step, but it's not at the start, and before the .targets.

https://learn.microsoft.com/en-us/visualstudio/msbuild/customize-by-directory?view=vs-2022